actioncable/lib/action_cable/connection/base.rb

```diff
@@ -272,6 +289,29 @@ def successful_request_message
             env["REQUEST_METHOD"], env["HTTP_CONNECTION"], env["HTTP_UPGRADE"]
           ]
         end
+
+        def expects_pong?
+          return @expects_pong unless @expects_pong.nil?
+
+          @expects_pong = @protocol&.starts_with?("actioncable-v1.1-")
+        end
+
+        def missed_pong_response?
+          expects_pong? && @last_message_received_at.before?(HALF_OPEN_CONNECTION_TIMEOUT.ago)
+        end
+
+        def handle_pong(payload)
+          latency = Time.now.to_f - payload["message"].to_f
+
+          if latency.positive?
+            ActiveSupport::Notifications.instrument(
```

Comment:
We don't. It's a way to expose the measured latency to the app so that it can be processed.

E.g. I track the latency of each Device and mark ones that have a latency of 1s or more as problematic for another team to look into.

Do you think it would be worth exposing the latency via a dedicated interface or do you think it shouldn't be exposed at all?

Reply:
I'm just trying to understand the goal. Maybe we should document this new instrumentation in the guide. https://guides.rubyonrails.org/active_support_instrumentation.html#action-cable

Reply:
Oh, I didn't know these were documented, thank you for pointing that out. 
I've updated the guide and change the instrumentation name to be in line with the other instrumentations.